use/events.js

```diff
@@ -240,12 +238,12 @@ export const useEvents = () => {
       name: 'price',
```

Comment:
Form field label if not set is default to upper-cased name, in this example it will be "Price".
If label is not set, please add it.